README.md

```diff
@@ -44,6 +44,31 @@ To run the following examples you will also need these ROS 1 packages:
 * `rostopic`
 * `rqt_image_view`
 
+### Prerequisites for the examples in this file
+
+In order to make the examples below portable between versions of ROS, we've chosen to define two environment variables, `ROS1_INSTALL_PATH` and `ROS2_INSTALL_PATH`. 
+These are defined as the paths to the installation location of their respective ROS versions.
+
+If you installed Noetic in the default location, then the definition of `ROS1_INSTALL_PATH` would likely be `/opt/ros/noetic`.
+
+Building the bridge as described below requires you to build all of ROS 2.
+We assume that you have downloaded it to `~/ros2_galactic`, and that is where you plan on building it.
```

Comment:
```suggestion
We assume that you have downloaded it to `~/ros2`, and that is where you plan on building it.
```

Reply:
What about mirroring what happens in `/opt/ros`?  That is, instead of `~/ros2_galactic`, the download location is `~/ros2/galactic`?  This lets end users download and install multiple versions of ROS 2 to their machines.

Reply:
Upon further reflection, and checking the existing documentation, we should keep the path as `~/ros2_galactic` because that is where the installation documentation says to put it.

Reply:
OK, I'll leave it as-is.  Thank you.